pkg/llbutils/utils.go

```diff
@@ -123,59 +123,159 @@ func Mkdir(state llb.State, owner string, dirs ...string) llb.State {
 	return state
 }
 
+// SystemPackagesCaching holds all the options relating to layer caching and
+// package caching.
+type SystemPackagesCaching struct {
+	IgnoreCache      bool
+	WithCacheMount   bool
+	CacheIDNamespace string
+}
+
+func NewCachingStrategyFromBuildOpts(
+	buildOpts builddef.BuildOpts,
+) SystemPackagesCaching {
+	return SystemPackagesCaching{
+		IgnoreCache:      buildOpts.IgnoreCache,
+		WithCacheMount:   buildOpts.EnableCacheMounts,
```

Comment:
why not to also name the `WithCacheMount` flag `EnableCacheMounts` ? (it would ease code lookups IMHO ^^)

Reply:
:+1:  and I think I'll rename `IgnoreCache` into `IgnoreLayerCache` to make it more meaningful :smile: